zaza/configure/bgp_speaker.py

```diff
@@ -22,6 +27,29 @@ def setup_bgp_speaker(peer_application_name, keystone_session=None):
     :returns: None
     :rtype: None
     """
+    # Get ASNs from deployment
+    dr_unit = zaza.model.get_first_unit_name(utils.get_juju_model(),
+                                             'neutron-dynamic-routing')
+    peer_unit = zaza.model.get_first_unit_name(utils.get_juju_model(),
+                                               peer_application_name)
+    dr_relation = yaml.load(
```

Comment:
Both of these really look like a utility begging to be created. Can we create a get_relation_data kind of utility in zaza.utilities._local_utils.py?

This will get reused again.